apps/web/src/app/(signing)/sign/[token]/form.tsx

```diff
@@ -30,6 +31,7 @@ export const SigningForm = ({ document, recipient, fields }: SigningFormProps) =
 
   const { fullName, signature, setFullName, setSignature } = useRequiredSigningContext();
 
+  const [showConfirmSignatureDialog, setShowConfirmSignatureDialog] = useState(false);
```

Comment:
Is there a reason why we lift this up from the `SignDialog` component?

It doesn't appear to be used anywhere in this parent component